deploy/cloudflare.md

```diff
@@ -12,34 +12,33 @@ outline: deep
 Uploading via Web UI is limited to 300 MB.
 
 Use [rclone](https://rclone.org/downloads/) to upload larger PMTiles archives to R2 (you can use the `rclone/rclone` docker image in order to avoid installation, the config is at `/etc/rclone` for mounting). 
-In order to use rclone you'll need to create an API key. Copy the "Access Key ID", the "Secret Access Key" and the "Endpoints for S3 clients" from the API key creation screen.
+This requires a token from **R2 > Manage R2 API Tokens**. Note **Access Key ID**, the **Secret Access Key** and the **Endpoint for S3 Clients** from the API key creation screen.
 Run the following commands:
-1. `rclone config` and follow the on screen questions, the endpoint should be something like: `https://<ACCOUNT_ID>.r2.cloudflarestorage.com/`
-2. `rclone copy your.pmtiles <the name you gave the above rclone configuration>:<BUCKET_NAME> --s3-no-check-bucket` to upload the file to the root of the bucket
+1. `rclone config` and follow the on screen questions, the endpoint should look like: `https://<ACCOUNT_ID>.r2.cloudflarestorage.com`
+2. `rclone copyto <FILE> <rclone configuration name>:<BUCKET_NAME>/<FILE> --progress --s3-no-check-bucket --s3-chunk-size=256M` to upload to the root of the bucket.
```

Comment:
Not sure the file at the end won't create a folder with the file name and place the file inside. I ran out without the FILE at the end.

Reply:
I just tested it like this:

```
% rclone copyto LICENSE.md tester:foobar/LICENSE.md  --s3-no-check-bucket --s3-chunk-size=256M --progress
Transferred:   	          0 B / 0 B, -, 0 B/s, ETA -
Elapsed time:         0.8s
% rclone ls tester:foobar
     3118 LICENSE.md
```

Reply:
How does it look in the cloudflare UI in the bucket?

Reply:
<img width="748" alt="Screenshot 2023-12-29 at 15 47 41" src="https://github.com/protomaps/docs/assets/77501/9b43ce85-7c97-4c4d-96c3-9d68cdd61c1b">

Reply:
Ok, cool.